iina/AutoFileMatcher.swift

```diff
@@ -45,9 +46,11 @@ class AutoFileMatcher {
 
     // group by extension
     for file in files {
-      let fileInfo = FileInfo(file)
-      if let mediaType = Utility.mediaType(forExtension: fileInfo.ext) {
-        filesGroupedByMediaType[mediaType]!.append(fileInfo)
+      autoreleasepool{
```

Comment:
Huh, in that case this isn't necessary. Is this thing applied to all loops? Forgive me cuz I'm mostly a objc guy.